apps/web/pages/api/registry/entries.ts

```diff
@@ -44,6 +46,12 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     },
   };
 
+  try {
+    const requests = await kv.get<number>(`stat:requests.${pageKey}`);
+    await kv.set<number>(`stat:requests.${pageKey}`, requests ?? 0 + 1);
```

Comment:
So we shouldn't need to get the key or do the start with zero logic if U use `INCRBY` https://redis.io/docs/latest/commands/incrby/